src/utils/shared/localStorage.js

```diff
@@ -0,0 +1,13 @@
+export const getDetails = () => {
+  let details = []
+  try {
+    details = JSON.parse(localStorage.getItem('details') || '[]')
+  } catch {}
```

Comment:
silent `catch` is bad? what kind of errors do we expect here?

Reply:
an error can occur when localStorage is not supported or it fails to parse what is coming back from localStorage which can happen if someone goes into localStorage and set the value for the key to something like `{`, ']' and so on i.e when there is invalid JSON. 

Since we are returning the empty array no matter what the app won't break.

I think adding a console.log would be helpful.